content/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling-part-2-streaming.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Discover the power of data center monitoring using Redfish telemetry and
+  cloud-native tooling: Part 2 - streaming"
```

Comment:
Capitalize the "S" in "Streaming". Because it's coming after "Part 2 -" it's actually the beginning of a title, so it gets to have a capital letter :-).

Reply:
Fixed